source/Nuget.nanoFramework.Hardware.ESP32/Nuget.nanoFramework.Hardware.ESP32.nuproj

```diff
@@ -0,0 +1,74 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Several fixes here:
- correct the Nuget name to nanoFramework.Hardware.ESP32
- the include files bellow should be referring to the ones from this project instead of Gpio ones
- the description should be descriptive for this package